extensions/wikia/CreateNewWiki/js/CreateNewWiki.js

```diff
@@ -275,6 +261,10 @@
 			this.answer = v;
 		},
 
+		onAuthSuccess: function () {
+			window.location.href = decodeURIComponent(this.redirectUrl).replace(/&/, '?');
```

Comment:
do we need to refresh a page?
Maybe we could just continue create new wiki flow?